obspy/clients/fdsn/client.py

```diff
@@ -475,6 +476,12 @@ def get_events(self, starttime=None, endtime=None, minlatitude=None,
         :type magnitudetype: str, optional
         :param magnitudetype: Specify a magnitude type to use for testing the
             minimum and maximum limits.
+        :type eventtype: str, optional
+        :param eventtype: Limit to events with a specified event type.
+            Multiple types are comma-separated (e.g.,
+            ``"earthquake,quarry blast"``). Allowed values are from QuakeML.
+            See :const:`obspy.core.event.header.EventType` for a list of
```

Comment:
The reason why I would like to bud +DOCS is that I'm not sure whether this link works

Reply:
I could build the docs locally thanks to the excellent work of by @heavelock https://github.com/heavelock/obspy/pull/9 (merci !) and verified that this works.

So, good to go!